src/plugins/github/fetchGithubRepo.js

```diff
@@ -35,9 +35,9 @@ import {cacheIdForRepoId} from "./cacheId";
  */
 export default async function fetchGithubRepo(
   repoId: RepoId,
-  options: {|+token: string, +cacheDirectory: string|}
+  options: {|+token: string, +cache: CacheProvider|}
```

Comment:
as elsewhere, i'd prefer that `cache` be called `cacheProvider`

Reply:
Using above cast instead.